locales/en/translation.json

```diff
@@ -294,6 +294,11 @@
   "dddd, MMMM D": "dddd, MMMM D",
   "dddd, h:mm a": "dddd, h:mm a",
   "delivered": "delivered",
+  "device-usage-dbl": "DBL",
```

Comment:
I find it more pratical when having this format:

```
"dbl":"DBL"
"devices": "Devices",
"deleted": "Deleted",
"last-updates": "Last updates",
"device-usage": "Device usage",
"show-more": "Show more",
"patient-statistics": "Patient statistics",
```
That way, the key in not coupled with any specific component and can be reused in any other component. Having the key being the closest looking possible to the values also helps in avoiding duplicates.
The value seems to always start with an uppercase too, so I advise keeping that rule for consistency.
Sorry if I'm being a bit annoying with this :stuck_out_tongue: